$OpenBSD: patch-git-cvsimport_perl,v 1.2 2007/07/24 05:16:04 bernd Exp $

Backport commit 7051c3b1294a110a862a7b48e983490fba32389f from the git
repository to fix a problem with git-cvsimport.

Fixed in git 1.5.3.x.

--- git-cvsimport.perl.orig	Thu Jul 12 19:24:26 2007
+++ git-cvsimport.perl	Thu Jul 12 19:24:54 2007
@@ -1020,7 +1020,7 @@ if ($orig_branch) {
 		unless -f "$git_dir/refs/heads/master";
 	system('git-update-ref', 'HEAD', "$orig_branch");
 	unless ($opt_i) {
-		system('git checkout');
+		system('git checkout -f');
 		die "checkout failed: $?\n" if $?;
 	}
 }
